### PR TITLE
Buffs the syndicate toolbox

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: GuyonBroadway/gorram_swags/Banthebantz
add: Added new tools to the syndicate toolbox, these are all twice as fast as normal tools and the crowbar has a special hidden xenophobic feature with a little help from Banthebantz.
imageadd: Sprites for these lovely new tools by gorram_swags.
/:cl:

The toolbox is a hardly ever purchased syndicate item. Sure it;s supposed to be "for emergencies" but on this server an emergency is always "imminent death" and it's incredibly rare you are ever in a situation where you need a set of tools right here right now where you couldn't have gotten some for free.

Also the bonus alien damage is revenge for all the times someone has had their antag round ruined by random aliens, or a half life reference, whatever you prefer.

pics

image

### CODE REVIEW

**Comment by PopNotes** [comment]
I like how the wirecutters are now chain-link fence cutters

File: hippiestation/code/game/objects/items/devices/multitool.dm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/obj/item/device/multitool/syndicate
+	name = "Black market multitool"
+	desc = "This multitool is a favourite among saboteurs, it's optimised routine lets it reset circuitry faster but is otherwise a normal multitool."
```
Comment (ghost) [inline hippiestation/code/game/objects/items/devices/multitool.dm]: it's is a contraction not possessive.
use "its"

File: hippiestation/code/game/objects/items/weapons/storage/tools.dm [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	name = "Pipe wrench"
+	desc = "A rather dangerous looking pipe wrench with teeth that grip better than a normal wrench."
+	icon = 'hippiestation/icons/obj/tools.dmi'
+	icon_state = "wrench_nuke"
+	origin_tech = "materials=1;engineering=1;syndicate=1"
+	toolspeed = 0.5
+
+/obj/item/weapon/wirecutters/syndicate
+	name = "Bolt cutters"
+	desc = "A sturdy set of bolt cutters, lets the user put more leverage into cutting through grilles, wires and people."
```
Comment (ghost) [inline hippiestation/code/game/objects/items/weapons/storage/tools.dm]: bad comma splicing
"A study set of bolt cutters wires and people"
change this to "A sturdy set of bolt cutters that lets the user put more leverage into cutting through grilles, wire and people."

**Comment by PopNotes** [comment]
`proc/gordon_freeman`
I love this

**Comment by KayeArray** [comment]
in the flesh or rather in the hazard suit

**Comment by PopNotes** [comment]
it's about to go _critical_

**Comment by TrickyDickRichard** [comment]
half life scientist outfit when

**Comment by KayeArray** [comment]
well if you mean the HEV suit I plan to port it soon

**Comment by TrickyDickRichard** [comment]
i mean the fucken SCIENTIST outfit. yknow white pants, blue shirt and red tie with dark blue stripes under a labcoat

**Comment by KayeArray** [comment]
WHY DO WE ALL HAVE TO WEAR THESE RIDICULOUS TIES?

File: hippiestation/code/game/objects/items/devices/multitool.dm (hunk 1 above)
Comment (tortellinitony) [reply hippiestation/code/game/objects/items/devices/multitool.dm]: start a NEW sentece

**Comment by PopNotes** [comment]
Oh, you wanna know why? I'll tell you why.
It's symbolic. The management wants you to know that you're their dog, so you're wearing their leash. You don't see me wearing a tie, do you? You know why? Because I'm a rebel. The day they tried to enforce the dress code on me, I let them know there could be an accident in this place if that happened. And by 'accident', I mean 'bomb the place.'

**Comment by ghost** [comment]
i'm retarded
i like this +1

**Review by tortellinitony — APPROVED** [review]

**Pull request merged**